gnomad/utils/transcript_annotation.py

```diff
@@ -0,0 +1,59 @@
+"""Utils module containing generic functions that are useful for adding transcript expression-aware annotations."""
+from typing import Callable, Optional
+
+import hail as hl
+
+
+def summarize_rsem_mt(
```

Comment:
can we also rename this to `summarize_transcript_expression`, then the field 'rsem' to 'transcript_expression', because I will have 'gene_expression' = sum of transcripts expression, then we'll take a proportion.